.vsts-ci.yml

```diff
@@ -15,154 +16,163 @@ pr:
     include:
     - '*'
 
-stages:
-- stage: build
-  displayName: Build
-  jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
-    parameters:
-      enableMicrobuild: true
-      enablePublishUsingPipelines: true
-      enablePublishBuildArtifacts: true
-      enablePublishTestResults: true
-      enableTelemetry: true
-      helixRepo: dotnet/cli-lab
-      timeoutInMinutes: 180 # increase timeout since BAR publishing might wait a long time
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: $(DncEngInternalBuildPool)
+      image: 1es-windows-2022-pt
```

Comment:
dnceng says the non -pt images have now been updated to support the template migration. You can switch back now or later (they said the -pt ones will go away eventually). Either way.